### PR TITLE
Add green glow hover effect to cards

### LINKED ISSUE #4986: [UI] No Hover Effect on Cards
### Description

These cards don't have the commonly used glowing hover effect on the rest of the site, which makes it look inconsistent.
Page link: https://layer5.io/learn/learning-paths

image

### Expected Behavior

We can add a border glow effect on hover to these cards as well, to make the design look uniform and even. 
image

### Environment:
- Host OS: Windows
- Browser: Chrome 

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Akshun-01**
**Description**
The Cards on the Learning Path page had no hover effect unlike the rest of the website
image

This PR fixes #4986 
Added the green hover effect to the cards, similar to the pricing page cards.
image

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit cba501f202c9b42bf54f8e64642891fc4daaf182 at: https://65246bf8d921243d068d2918--layer5.netlify.app

File: src/components/Learn-Components/Card-Component/learn-card.style.js [hunk 1]
```diff
@@ -5,6 +5,13 @@ const CardWrapper = styled.div`
     min-height: 16rem;
     margin: auto;
     border-radius: 1rem;
+    cursor: pointer;
+    box-shadow: rgba(0, 179, 158, 0.9) 0px 2px 10px 1px;
```
Comment (sudhanshutech) [inline src/components/Learn-Components/Card-Component/learn-card.style.js:9]: changes looks good, but the box shadow does not have the perfect curve effect at corners if you see
image

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: src/components/Learn-Components/Card-Component/learn-card.style.js (hunk 1 above)
Comment (Akshun-01) [reply src/components/Learn-Components/Card-Component/learn-card.style.js:9]: ok I'll look into it

**Comment by Yashsharma1911** [comment]
@Akshun-01 checking in, any update?

**Comment by Akshun-01** [comment]
Yeah, sorry for the delay, was busy with some college work.
This is the latest one, I altered the glow effect so that it looks better with curved edges as requested. You can review the changes

 

https://github.com/layer5io/layer5/assets/93793691/1ee04228-9537-4c04-ba32-44abadc7b617

**Comment by l5io** [comment]
🚀 Preview for commit 6ef47e6e59ed214907ecbec183e2b76c215ef501 at: https://652bed3503f1d66bc428e4b9--layer5.netlify.app

**Comment by ritiksaxena124** [comment]
@Akshun-01 the changes look great. Let's only show the green box shadow on the hover of the cards, not always. Also not show a box shadow on the "Mastering Service Meshes for Operators" card as it is not currently clickable.

**Comment by Akshun-01** [comment]
Hey @ritiksaxena124, I made the required changes.


https://github.com/layer5io/layer5/assets/93793691/720ae589-df48-4968-8d44-2dc7191bbf8a

**Comment by Akshun-01** [comment]
Hey, can anyone tell me why the checks failed this time? I made sure not to change any unrelated files.

**Comment by l5io** [comment]
🚀 Preview for commit e7aa89bb7fdfc34f70138703d4f2ea46f1818075 at: https://652c253b412cb31afa49ed9c--layer5.netlify.app

**Comment by saurabh100ni** [comment]
Thanks for raising PR, Let's discuss this in the Website call today at 5:30 PM IST.
Please add this as your agenda in [Docs](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk)

**Review by abhijeetgauravm — COMMENTED** [review]
Looks good to me!

**Comment by Akshun-01** [comment]
@ritiksaxena124 are any other changes required in this?

**Review by ritiksaxena124 — APPROVED** [review]
LGTM 🙌

**Pull request merged**